ngraph/core/src/descriptor/tensor.cpp

```diff
@@ -90,13 +85,41 @@ size_t descriptor::Tensor::size() const
     return shape_size(get_shape()) * m_element_type.size();
 }
 
+NGRAPH_SUPPRESS_DEPRECATED_START
+void descriptor::Tensor::set_name(const string& name)
+{
+    m_name = name;
```

Comment:
should not it insert name into `m_names` map as well?

Reply:
yes we can I think.
At the current moment we use set/get_name only for TensorIterator case.

Reply:
I think we shouldn't add legacy tensor name which we use in the TI in order to avoid additional risks with name collision.